buildAndRunSite.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-jekyll serve --incremental --watch
\ No newline at end of file
+jekyll serve --incremental --watch --config _config.yml,_config_en.yml 
```

Comment:
The readme refers to running the buildAndRunSite.sh, but we will be changing the way we build the two sites, so not yet.